guardrails/guard.py

```diff
@@ -34,6 +34,7 @@
 from langchain_core.messages import BaseMessage
 from langchain_core.runnables import Runnable, RunnableConfig
 from pydantic import BaseModel
+from pydantic.version import VERSION as PYDANTIC_VERSION
```

Comment:
You can consolidate this as
```
from pydantic import BaseModel, version as PYDANTIC_VERSION
```

Reply:
linter hates this!! I agree it looks better, but it fails lint

Reply:
kk let's merge as is then